monai/apps/detection/transforms/box_ops.py

```diff
@@ -318,3 +318,31 @@ def convert_mask_to_box(
     boxes, *_ = convert_to_dst_type(src=boxes_np, dst=boxes_mask, dtype=box_dtype)
     labels, *_ = convert_to_dst_type(src=labels_np, dst=boxes_mask, dtype=label_dtype)
     return boxes, labels
+
+
+def select_labels(
+    labels: Union[Sequence[NdarrayOrTensor], NdarrayOrTensor], keep: NdarrayOrTensor
+) -> Union[Tuple, NdarrayOrTensor]:
+    """
+    For element i in labels, select labels[i][keep]
```

Comment:
Could you please help add more description for the function?
I feel this doc-string is almost the same as one line source code..`For element i in labels, select labels[i][keep]`

Thanks.